src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/View/products.html.twig

```diff
@@ -133,7 +133,7 @@
           </ul>
         </nav>
       </div>
-      <div class="col-md-6 text-right discount-action">
+      <div class="col-xl-8 text-right discount-action">
```

Comment:
Shouldn't it be `col-md-8` ? This means they won't have the same behavior depending on the screen size?

Reply:
This is to fix one of points of this PR